### PR TITLE
[totem-pl-parser] Update to 3.26.6. JB#56421

### PR DESCRIPTION
**Pull request by pvuorela**
And Gmime no longer needed.

@Tomin1 @mlehtima

### CODE REVIEW

File: rpm/totem-pl-parser.spec [hunk 1]
```diff
@@ -1,14 +1,12 @@
 Name:       totem-pl-parser
 Summary:    Totem Playlist Parser library
-Version:    3.26.1
+Version:    3.26.6
 Release:    1
-Group:      System/Libraries
 License:    LGPLv2+
 URL:        http://www.gnome.org/projects/totem/
 Source0:    http://download.gnome.org/sources/%{name}/3.26/%{name}-%{version}.tar.bz2
```
Comment (saukko) [inline rpm/totem-pl-parser.spec:7]: Extra points: This should be %{name}-%{version}.tar.bz2 as the upstream source tarball is not the one we use here directly.

**Comment by saukko** [comment]
Extra points from adding -Denable-libgcrypt=true -Denable-libarchive=false to ensure build does not change based on build environment. For the dropped gmime that was extra dep to start with there is no meson config, right?

File: rpm/totem-pl-parser.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/totem-pl-parser.spec]: Sure. Updated.

**Comment by pvuorela** [comment]
Added the explicit build flags.

GMime was used by the earlier version for one function call or something like that. but on this new one that small part of the code instead having a local copy.

**Review by mlehtima — APPROVED** [review]

**Pull request merged**